package.json

```diff
@@ -29,7 +29,8 @@
     "_typings-dts-bundle": "node ./node_modules/dts-bundle/lib/dts-bundle.js --main dist/dts/index.d.ts --name ontodia --out ../ontodia.d.ts",
     "_typings": "npm run _typings-tsc && npm run _typings-dts-bundle",
     "test": "npm run build",
-    "_webpack": "node ./node_modules/webpack/bin/webpack.js"
+    "_webpack": "node ./node_modules/webpack/bin/webpack.js",
+    "start" : "node ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --port 5000 --host 0.0.0.0 --config webpack.demo.config.js --content-base dist/"
```

Comment:
Start without arguments is needed to run demos on heroku/dokku. Could introduce additional arguments for demo and call 'npm run demo args' from start, but there are to few args to add and reuse.